mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -861,7 +861,7 @@ prefs_dialog.mail_address = Your email address
 prefs_dialog.mail_server = SMTP server
 prefs_dialog.misc_tab = Misc
 prefs_dialog.confirm_on_quit = Show confirmation dialog on quit
-prefs_dialog.shell = $[RunCommand.label]
+prefs_dialog.shell = Shell
```

Comment:
how about calling it "Terminal" then to reduce the amount of translation items?

Reply:
Afair, this is for choosing bash vs zsh, so shell is appropriate I think

Reply:
> how about calling it "Terminal" then to reduce the amount of translation items?

ok, now I'm looking normally thru laptop - yes, you are right. I will change this to Terminal

Reply:
@ahadas  PR updated